With BKL <smp_lock.h> gone away from linux.

Patch fixes build failure against stable kernel:
    CC [M]  kqemu-1.4.0pre1/kqemu-linux.o
    kqemu-1.4.0pre1/kqemu-linux.c:27:28: error: linux/smp_lock.h: No such file or directory

Bug-URL:  http://bugs.gentoo.org/368439
Reported-by: Ian Abbott
Patch-by: c.exe@gmx.de

diff --git a/kqemu-linux.c b/kqemu-linux.c
index f7246f1..c476d28 100644
--- a/kqemu-linux.c
+++ b/kqemu-linux.c
@@ -24,7 +24,9 @@
 #include <linux/proc_fs.h>
 #include <linux/version.h>
 #include <linux/ioctl.h>
+#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,39)
 #include <linux/smp_lock.h>
+#endif
 #include <linux/miscdevice.h>
 #include <asm/atomic.h>
 #include <asm/processor.h>
